docs/sync-rules/redis.md

```diff
@@ -0,0 +1,103 @@
+### Setting up the Redis server/cloud connector
+
+See the [Developer guide](../../docs/DEVELOPING.md) for setting up connectors.
+
+### Overview
+
+Advanced Sync Rules help manage data in Redis server/cloud. They take the following parameters:
+
+1. `database`: Database indice of Redis server/cloud. Type of value should be integer.
+2. `key_pattern`: Pattern of the key for searching in Redis server/cloud
+3. `_type`: Type of key(i.e. HASH, LIST, SET, STREAM, STRING, ZSET) in Redis server/cloud.
+
+*NOTE*:
+
+- `key_pattern` or `_type` is not required but any one of them is required to pass.
```

Comment:
I think this needs to be re-phrased. cc @leemthompo WDYT?

Reply:
Agree!